apps/core/src/components/root-app-sidebar/index.tsx

```diff
@@ -111,6 +111,7 @@ export const RootAppSidebar = ({
     if (isDesktop) {
       return window.events?.applicationMenu.onNewPageAction(onClickNewPage);
     }
+    return () => {};
```

Comment:
noop from foxact/noop

Reply:
It's type is 
```ts
interface Noop {
    (...args: any[]): any;
}
```
and ts does not like that